README.md

```diff
@@ -9,14 +9,15 @@ A web forum demo application built on OrbisDB.
 This application emulates a web forum, complete with posts, comments, images, and personal profiles. It relies on the following items:
 
 **OrbisDB on Ceramic** - A decentralized database to hold the various forum artifacts users create (such as posts, comments, etc.)
+**TACo** - A decentralized access control layer, through which decryption rights to non-public forum content (posts, comments, reactions, etc.) are verified and enforced by a decentralized cohort of TACo nodes. 
 **ThirdWeb SDK** - Uses their IPFS upload feature for image file storage
 
 ## Installation
 
 First, create a local clone of the application:
 
 ```bash
-git clone https://github.com/ceramicstudio/deforum && cd deforum
+git clone https://github.com/nucypher/taco-orbisdb-demo && cd taco-orbisdb-demo
```

Comment:
```suggestion
git clone https://github.com/nucypher/deforum-taco && cd deforum-taco
```

Reply:
Actually, if this is going to be merged eventually on `ceramicstudio/deforum-taco`, should not be this the URL?

`git clone https://github.com/nucypher/deforum-taco`

Reply:
Good point - although the README in `nucypher/deforum-taco` probably shouldn't say that.

Perhaps it is better to make this more flexible and just say something akin "clone this repos".

Reply:
done